lib/ssh/doc/src/ssh.xml

```diff
@@ -1107,6 +1107,88 @@
      </desc>
    </func>
 
+<!-- direct_tcpip/5 -->
+   <func>
+     <name name="direct_tcpip" arity="5" since=""/>
+     <fsummary>Forward local port to a server.</fsummary>
+     <desc>
+       <p>Listen on <c>LocalHost</c>:<c>LocalPort</c> and forward connections to <c>RemoteHost</c>:<c>RemotePort</c>. 
+       </p>
+       <p>Returns a port assigned on the client.</p>
+       <p> <c>LocalHost</c> interpretation:</p>
+       <list>
+         <item><c>"0.0.0.0"</c> means to listen on all IPv4 addresses</item>
+         <item><c>"::"</c> means to listen on all IPv6 addresses</item>
+         <item><c>loopback</c> means to listen on loopback interface</item>
+         <item><c>"127.0.0.1"</c> and <c>"::1"</c> indicate listening on the loopback
+         interfaces for IPv4 and IPv6, respectively</item>
+         <item>other values are interpreted as usually</item>
+       </list>
+       <p>If <c>LocalPort</c> is 0, a listener is assigned to an arbitrary free port</p>
+     </desc>
+   </func>
+<!-- tcpip_forward/5,6 -->
+   <func>
+     <name since="">tcpip_forward(ConnectionRef, RemoteHost, RemotePort, LocalHost, LocalPort) -> Result</name>
+     <name since="">tcpip_forward(ConnectionRef, RemoteHost, RemotePort, LocalHost, LocalPort, Timeout) -> Result</name>
+     <fsummary>Forward server's port to a client.</fsummary>
+     <type>
+       <v> ConnectionRef = <seealso marker="#type-connection_ref">connection_ref()</seealso></v>
+       <v>RemoteHost = <seealso marker="#type-host">host()</seealso></v>
+       <v>RemotePort = <seealso marker="kernel:inet#type-port_number">inet:port_number()</seealso></v>
+       <v>LocalHost = <seealso marker="#type-host">host()</seealso></v>
+       <v>LocalPort = <seealso marker="kernel:inet#type-port_number">inet:port_number()</seealso></v>
+       <v>Timeout = timeout()</v>
+       <v>Result = {ok, <seealso marker="kernel:inet#type-port_number">inet:port_number()</seealso>} | {error, term()}</v>
+     </type>
+     <desc>
+       <p>Ask the server to listen on <c>RemoteHost</c>:<c>RemotePort</c> and forward connections to <c>LocalHost</c>:<c>LocalPort</c>. 
+       </p>
+       <p>Returns a port assigned on the server.</p>
+       <p> <c>RemoteHost</c> interpretation:</p>
+       <list>
+         <item><c>"0.0.0.0"</c> means to listen on all IPv4 addresses</item>
+         <item><c>"::"</c> means to listen on all IPv6 addresses</item>
+         <item><c>loopback</c> means to listen on loopback interface</item>
+         <item><c>"127.0.0.1"</c> and <c>"::1"</c> indicate listening on the loopback
+         interfaces for IPv4 and IPv6, respectively</item>
+         <item>other values are interpreted as usually</item>
+       </list>
+       <p>If <c>RemotePort</c> is 0, a listener is assigned to an arbitrary free port</p>
+     </desc>
+   </func>
+   
+   <func>
```

Comment:
The connect fun is already defined in lines 1069 - 1098

Reply:
Thank you fir quick response,
fixed